ONSdigital/dp-frontend-search-controller
# topic cache refactor and update fix

### What

Cache update fix and refactored data topic cache impl

### How to review

- Check code is ok
- Pull branch
- set EnableAggregationPages and EnableTopicAggregationPages to 'true'
- run application and try accessing a route with topic, subtopic and subsubtopic in its url from your browser
- Ensure correct data is displayed and cache working.

### Who can review

Not me.
